src/EnergyPlus/PlantChillers.cc

```diff
@@ -6539,6 +6539,10 @@ namespace PlantChillers {
                                             state.dataIPShortCut->cAlphaArgs(8)));
                 }
             }
+
+            // set default design condenser in and evaporator out temperatures
+            thisChiller.TempDesCondIn = 29.44; // Degree Celsius, or 85 Degree Fahrenheit
+            thisChiller.TempDesEvapOut = 6.67; // Degree Celsius, or 44 Degree Fahrenheit
```

Comment:
That should do it!

![image](https://github.com/NREL/EnergyPlus/assets/5778156/ea531960-11d8-4247-8c6a-bc65774f4d6b)

Reply:
@rraustad @jcyuan2020 - This appears to be from AHRI 550/590, the IP version of the standard. Curiously, AHRI 551/591, the SI version of the standard, has slightly different rated conditions (when you convert the temperature from deg. C to deg. F). Maybe there should be a note about it.

Here's same table but in 551/591, check out the leaving temperature (7 deg. C, not quite 6.67 deg. F):

![image](https://github.com/NREL/EnergyPlus/assets/5149279/7b736e7f-2c6b-4ed9-badb-cc27e97a6458)